managed/services/management/backup/backups_service_test.go

```diff
@@ -41,28 +41,37 @@ import (
 	"github.com/percona/pmm/managed/utils/tests"
 )
 
-func setup(t *testing.T, q *reform.Querier, serviceName string) *models.Agent {
+func setup(t *testing.T, q *reform.Querier, serviceType models.ServiceType, serviceName string) *models.Agent {
 	t.Helper()
+	isSupportedService := func() bool {
+		return serviceType == models.MySQLServiceType || serviceType == models.MongoDBServiceType
+	}
+	require.True(t, isSupportedService())
 	node, err := models.CreateNode(q, models.GenericNodeType, &models.CreateNodeParams{
-		NodeName: "test-node",
+		NodeName: "test-node-" + t.Name(),
 	})
 	require.NoError(t, err)
 
 	pmmAgent, err := models.CreatePMMAgent(q, node.NodeID, nil)
 	require.NoError(t, err)
 	require.NoError(t, q.Update(pmmAgent))
 
-	mysql, err := models.AddNewService(q, models.MySQLServiceType, &models.AddDBMSServiceParams{
+	var service *models.Service
+	service, err = models.AddNewService(q, serviceType, &models.AddDBMSServiceParams{
 		ServiceName: serviceName,
 		NodeID:      node.NodeID,
 		Address:     pointer.ToString("127.0.0.1"),
-		Port:        pointer.ToUint16(3306),
+		Port:        pointer.ToUint16(60000),
```

Comment:
Why?

Reply:
3306 is mysql port, using it here makes it seem like it's supposed to be a MySQL service which is misleading imo.